universe/Planet.h

```diff
@@ -1,6 +1,7 @@
 #ifndef _Planet_h_
 #define _Planet_h_
 
+#include "Enums.h"
```

Comment:
I'm a bit more hesitant with including `Enums.h` in unnecessary places because it can be changed for some many unrelated reasons. I suppose there's a difference between "makes lots of stuff need a recompile" vs. "makes recompiling lots of stuff slower", though.